### DISCUSSION TITLE
Warning: On entry to DLASCLS parameter number  4 had an illegal value

### DISCUSSION THREAD

**Discussion by thilnguy** [OpenStitching/stitching#205] [General]
Hello,
I sometimes get**On entry to DLASCLS parameter number  4 had an illegal value** in warp image step.
Therefore, the stitching alg always fails.
Here is match_graph:
`graph matches_graph{
"1" -- "2"[label="Nm=35, Ni=32, C=1.72973"];
"1" -- "3"[label="Nm=34, Ni=30, C=1.64835"];
"1" -- "4"[label="Nm=33, Ni=32, C=1.78771"];
"1" -- "5"[label="Nm=33, Ni=32, C=1.78771"];
"1" -- "6"[label="Nm=28, Ni=26, C=1.58537"];
"1" -- "7"[label="Nm=34, Ni=31, C=1.7033"];
"1" -- "8"[label="Nm=33, Ni=30, C=1.67598"];
"1" -- "9"[label="Nm=33, Ni=31, C=1.73184"];
"1" -- "10"[label="Nm=30, Ni=27, C=1.58824"];
"1" -- "11"[label="Nm=29, Ni=28, C=1.67665"];
"1" -- "12"[label="Nm=27, Ni=25, C=1.5528"];
"1" -- "13"[label="Nm=29, Ni=27, C=1.61677"];
"1" -- "14"[label="Nm=27, Ni=25, C=1.5528"];
"1" -- "15"[label="Nm=28, Ni=27, C=1.64634"];
"1" -- "16"[label="Nm=33, Ni=33, C=1.84358"];
"1" -- "17"[label="Nm=32, Ni=31, C=1.76136"];
"1" -- "18"[label="Nm=34, Ni=32, C=1.75824"];
"1" -- "19"[label="Nm=32, Ni=31, C=1.76136"];
"1" -- "20"[label="Nm=32, Ni=31, C=1.76136"];
}`
it also looks weird for me. Do you have any idea what may be the root of problem?

Thank you

>

**Comment by lukasalexanderweber** [comment]
Have you checked https://github.com/OpenStitching/stitching/discussions/159

**Reply by thilnguy** [reply]
Yes, Here are my configuration: 
                    detector: sift
                    confidence_threshold: 0.2
                    match_conf: 0.5
                    wave_correct_kind: vert
but I still have issue.
Here some photos:
![14](https://github.com/OpenStitching/stitching/assets/48985195/2f19acbf-7b6a-4b5e-b01a-dc8f7bb2aad3)
![15](https://github.com/OpenStitching/stitching/assets/48985195/91206b0b-07ee-42a9-827c-2a9159f3c383)
![16](https://github.com/OpenStitching/stitching/assets/48985195/6fbfefdd-482c-4d15-b285-f87db036c740)
![17](https://github.com/OpenStitching/stitching/assets/48985195/5b231365-779a-4e0c-b908-eeb773b3569f)
![18](https://github.com/OpenStitching/stitching/assets/48985195/fa04c1bc-f528-4bf9-a718-0f6e3e4253c5)
Could you please take a look?
Thank you

**Reply by lukasalexanderweber** [reply]
This is out of scope of the discussions section

**Reply by thilnguy** [reply]
Sorry for my continue asking. What do you mean "out of scope"? 
as you suggest in https://github.com/OpenStitching/stitching/discussions/159, i add the **wave_correct_kind = vert**. However, i still have the warning **On entry to DLASCLS parameter number 4 had an illegal value**
Thank you

**Reply by lukasalexanderweber** [reply]
Sorry I just meant that I'll not have a look. But you are free to share your sollution in case others have the same problem

**Reply by lukasalexanderweber** [reply]
Maybe its a Windows Problem? https://stackoverflow.com/questions/64698111/what-causes-on-entry-to-dlascls-parameter-number-4-had-an-illegal-value-error
